src/redux/todo/reducers/operations.js

```diff
@@ -13,6 +14,11 @@ export const operationsReducer = (state = defaultState, action) =>{
         {
             return [...state,action.payload];
         }
+        case DELALL:
+        {
+            return [];
+        }
+
```

Comment:
you dont need to give body braces if it contains only line. Hence it will look more better.
Like this:
`const defaultState = [
  { id: 23, task: "Jomoloko", status: false },
];

export const operationsReducer = (state = defaultState, action) => {
  switch (action.type) {
    case ADD:
      return [...state, action.payload];
    case DELALL:
      return [];
    default:
      return state;
  }
};`